buildenv/jenkins/jobs/infrastructure/Cleanup-Nodes

```diff
@@ -169,19 +169,23 @@ timeout(time: TIMEOUT_TIME.toInteger(), unit: TIMEOUT_UNITS) {
                                     }
 
                                     // Clean up defunct pipelines workspaces
-                                    dir("${buildWorkspace}/../") {
```

Comment:
What is the current directory without this `dir() {}`? If it is within a directory we want to delete, it will fail on Windows (at least).

Reply:
The current directory is the pipeline build workspace. 
This is a work-around for the clean workspace plugin failures on windows. The code deletes old pipelines build workspaces that have not been deleted by the cleanWs via `rm -rf ...` command, in the current directory( ==buildWorkspace).